components/runtime/node6/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2015 EveryCity Ltd. All rights reserved.
+# Copyright 2016 Andreas Wacknitz
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		node
+COMPONENT_FMRI=		runtime/nodejs-6
+COMPONENT_SUMMARY=	Evented I/O for V8 JavaScript.
+COMPONENT_CLASSIFICATION=	Development/Other Languages
+COMPONENT_PROJECT_URL=	http://nodejs.org
+COMPONENT_VERSION=	6.1.0
+COMPONENT_LICENSE=	BSD-like, MIT, Apache
+COMPONENT_LICENSE_FILE=	node.license	
+COMPONENT_SRC=		$(COMPONENT_NAME)-v$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:9e67ef0b8611e16e6e311eccf0489a50fe76ceebeea3023ef4f51be647ae4bc3
+COMPONENT_ARCHIVE_URL=	http://nodejs.org/dist/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+# Include /usr/sbin for dtrace
+PATH=/usr/bin:/usr/gnu/bin:/usr/sfw/bin:/usr/ccs/bin:/usr/sbin
+
+CONFIG_SHELL =	/usr/bin/python
```

Comment:
Don't do this, use $(PYTHON) or $(PYTHON.VERSION), as /usr/bin/python is mediated